src/components/PopupWithForm.js

```diff
@@ -39,12 +40,12 @@ export default class PopupWithForm extends Popup {
     });
   }
 
-  renderSaving(saving) {
+  renderSaving(saving, phrase = "Saving...") {
     if (saving) {
-      this._submitButton.textcontent = "Saving...";
+      this._submitButton.textcontent = phrase;
```

Comment:
`textcontent` should be changed to `textContent` in `camelCase` to make the loading effect work